install_config/install/deploy_router.adoc

```diff
@@ -99,6 +99,15 @@ communication between OpenShift components is secured by TLS and uses various
 certificates and authentication methods. Use the `--credentials` option to
 specify what credentials the router should use to contact the master.
 
+[IMPORTANT]
+====
+Attach routers directly to port 80 and 443 on all interfaces on a host. Restrict
```

Comment:
The user doesn't attach the routers -- the routers attach themselves.

Perhaps:

"Routers directly attach to port 80 and 443 on all interfaces on the hosts they run on."

?